examples/FIDtoEcho_Actual_Challenging.py

```diff
@@ -70,11 +67,16 @@
         fl.image(data["t2":(-1e3, 1e3)], ax=ax_list[0])
         ax_list[0].set_title("Raw Data")
         data = data["t2":f_range]
+        fl.next("Data processing", fig=fig)
         fl.basename = "(%s)" % label
-        data = fid_from_echo(data, signal_pathway, fl=fl)
+        data = fid_from_echo(
+            data,
+            data.get_prop("coherence_pathway"),
+            fl=fl,
+        )
```

Comment:
w/ my other comment, this would not change

Reply:
for that data that's used here, you can't pull the coh pathway b/c it's not set in the data

Reply:
it's set in the preprocessing